lib/rummager_finder_publisher.rb

```diff
@@ -9,20 +9,30 @@ def initialize(metadatas, logger: Logger.new(STDOUT))
 
   def call
     metadatas.each do |metadata|
-      if metadata[:file].has_key?("content_id")
+      if !preview_only?(metadata)
         export_finder(metadata)
-      else
-        # Even though rummager doesn't use the content_id we only want to push
-        # to rummager if this is live in content-store, so this needs to
-        # replicate the logic in PublishingApiFinderPublisher.
-        logger.info("didn't publish #{metadata[:file]["name"]} because it doesn't have a content_id")
+      elsif preview_only?(metadata)
+        if preview_domain_or_not_production?
+          export_finder(metadata)
+        else
+          puts "didn't publish #{metadata[:file]["name"]} because it is preview_only" if @log
```

Comment:
We should use `logger` instead of `puts` consistently in this class now (it was only added here recently).